doc/source/installation.rst

```diff
@@ -1,140 +1,150 @@
+To install the latest stable version of MDAnalysis via `mamba`_ with all dependencies for full functionality, use the following command. 
 
 .. code-block:: bash
 
     mamba install -c conda-forge mdanalysis
 
-To upgrade:
+To upgrade use:
 
 .. code-block:: bash
 
     mamba update mdanalysis
 
-To install the tests:
+To install the :ref:`test suite <mdanalysistests>` use:
 
 .. code-block:: bash
 
     mamba install -c conda-forge MDAnalysisTests
 
-If you intend to use MDAnalysis in JupyterLab, you will have to install
+If you intend to use MDAnalysis in **JupyterLab**, you will have to install
 an extra package for the progress bar in analysis classes:
 
 .. code-block:: bash
 
     conda install -c conda-forge nodejs
     jupyter labextension install @jupyter-widgets/jupyterlab-manager
 
-pip
-=====
-The following command will install or upgrade the latest stable version of MDAnalysis via ``pip``, with core dependencies. This means that some packages required by specific analysis modules will not be installed.
+.. _pip-installation:
+
+pip 
+===
+
+The following command will install or upgrade the latest stable version of MDAnalysis via `pip`_ with core dependencies. 
+This means that some packages required by specific analysis modules will not be installed.
 
 .. code-block:: bash
 
     pip install --upgrade MDAnalysis
 
-If you need to install a fully-featured MDAnalysis, add the ``analysis`` tag. As with ``conda``, this will not install external programs such as `HOLE`_.
+
+If you need to install a fully-featured MDAnalysis, add the ``analysis`` tag. 
 
 .. code-block:: bash
 
     pip install --upgrade MDAnalysis[analysis]
 
-To install/upgrade tests:
+To install or upgrade the :ref:`test suite <mdanalysistests>`:
 
 .. code-block:: bash
 
     pip install --upgrade MDAnalysisTests
 
-If you intend to use MDAnalysis in JupyterLab, you will have to install
+If you intend to use MDAnalysis in **JupyterLab**, you will have to install
 an extra package for the progress bar in analysis classes:
 
 .. code-block:: bash
 
     pip install nodejs
     jupyter labextension install @jupyter-widgets/jupyterlab-manager
 
+.. _source-installation:
 
-Development versions
-====================
-To install development versions of MDAnalysis, you can compile it from source. In order to install from source, you will need ``numpy`` and ``cython``. See :ref:`create-virtual-environment` for instructions on how to create a full development environment.
+Install from source
+===================
 
+If you plan to develop or contribute to MDAnalysis, follow these instructions first. Then, refer to :ref:`contributing` for additional guidelines.
+
+To install the latest development versions of MDAnalysis (including unreleased features and bug fixes) you can compile it from `source`_. Use the following commands:
```

Comment:
Our "from source" instructions are not very good because the **assuming you have already installed required dependencies** really shifts *all* the responsibility on the user to maintain a clean environment. 

We had discussions about shipping a conda env file but we have not achieved any kind of consensus there so the most accurate instructions are https://userguide.mdanalysis.org/stable/contributing_code.html#with-conda-forge-packages and then https://userguide.mdanalysis.org/stable/contributing_code.html#with-mamba-conda